Running-Mastodon/PgBouncer-guide.md

```diff
@@ -48,24 +48,24 @@ PgBouncer has two config files: `pgbouncer.ini` and `userlist.txt` both in `/etc
 
 Add the `mastodon` user to the `userlist.txt`:
 
-    "mastodon" "md599dff7ae01ae2dc33ae052264bf22bf4"
+    "mastodon" "md5d75bb2be2d7086c6148944261a00f605"
 
 Here we're using the md5 scheme, where the md5 password is just the md5sum of `username + password` with the string `md5` prepended. For instance, to derive the hash for user `mastodon` with password `password`, you can do:
```

Comment:
There seems to be a missing fix.
md5sum of `username + password` to  md5sum of `password + username`